lib/LedgerSMB/I18N.pm

```diff
@@ -135,6 +136,28 @@ sub get_country_list {
     ];
 }
 
+=item get_language_list($language)
+
+Get a language localized list to allow user selection
+
+=cut
+
+sub get_language_list {
+    my ($self,$language) = @_;
```

Comment:
In:

 https://github.com/ledgersmb/LedgerSMB/pull/6289/files#diff-535e82bbb947ea4d6c0b06744fcd4b564650c37c2adec048c531743dd014baa4R240

you're calling this routine, but not as `$i18n->get_language_list()`, but as `LedgerSMB::I18N::get_language_list( $report, $language );` -- if that's not an error then calling the report instance `$self` here is confusing.